packages/main/src/Tokenizer.ts

```diff
@@ -639,7 +649,7 @@ class Tokenizer extends UI5Element {
 			content: {
 				"ui5-tokenizer--content": true,
 				"ui5-tokenizer-expanded--content": !this.showNMore,
-				"ui5-tokenizer-nmore--content": this.showNMore,
+				"ui5-tokenizer-nmore--content": !!this.showNMore,
```

Comment:
`this.showMore` should always return a boolean, better change the getter?

Reply:
Done